packages/sources/react/src/components/VtmnTextInput/VtmnTextInput.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import clsx from 'clsx';
+
+import '@vtmn/css-text-input';
+
+export interface VtmnTextInputProps
+  extends React.InputHTMLAttributes<HTMLInputElement> {
+  /**
+   * Label text linked to the input
+   * @type {string}
+   */
+  label: string;
+
+  /**
+   * Helper text to help the user
+   * @type {string}
+   */
+  helperText?: string;
+
+  /**
+   * Whether input is successful or not
+   * @type {boolean}
+   */
+  valid?: boolean;
+
+  /**
+   * Whether input is in error or not
+   * @type {boolean}
+   */
+  error?: boolean;
+}
```

Comment:
Because we have chosen to have an `id` required, it could be nice to see it there as a prop types with TSDoc that explains it (because we use the "for" attribute of the required `<label>`)

Reply:
On the ID prop ? Not here ?

Reply:
? Just declare the `id` prop inside VtmnTextInputProps.